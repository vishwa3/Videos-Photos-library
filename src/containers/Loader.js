import React from 'react';

function ShowDetail() {
  return (
    <div className="loader centre">
   {/*    <i className="fa fa-cog fa-spin" /> */}
      {/*  <i className="fas fa-spinner fa-spin" />  */}
       <i className="fas fa-yin-yang fa-spin" />
    </div>
  );
}

export default ShowDetail;