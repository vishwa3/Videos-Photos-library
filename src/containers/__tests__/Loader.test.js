import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../Loader';

describe('loader',() => {
  test('snapshot for Loader', () => {
    const component = renderer.create(<Loader />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})