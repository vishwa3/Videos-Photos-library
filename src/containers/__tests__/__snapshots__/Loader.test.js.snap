// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loader snapshot for Loader 1`] = `
<div
  className="loader centre"
>
  <i
    className="fas fa-yin-yang fa-spin"
  />
</div>
`;
